shared/common-adapters/section-list.d.ts

```diff
@@ -7,6 +7,20 @@ type DesktopProps = {
   selectedIndex: number | undefined
 }
 
+export interface SectionBase<ItemT> {
+  data: ReadonlyArray<ItemT>
+
+  key?: string
+
+  renderItem?: ({index: number, item: ItemT}) => React.ReactNode | null
+
+  ItemSeparatorComponent?: React.ComponentType<any> | null
+
+  keyExtractor?: (item: ItemT, index: number) => string
```

Comment:
Why so much whitespace between the lines?

Reply:
And where are `ItemSeparatorComponent` and `keyExtractor` used? I can't find them anywhere in this PR.

Reply:
I stole this type from React Native, not sure why they use extra lines. Those aren't used but they are part of the type on mobile only. I removed them from this type, you're right that they aren't necessary and probably they create confusion.